cmd/exchange_wrapper_standards/exchange_wrapper_standards_test.go

```diff
@@ -645,6 +645,16 @@ var acceptableErrors = []error{
 	limits.ErrExchangeLimitNotLoaded,     // Is thrown when the limits aren't loaded for a particular exchange, asset, pair
 	limits.ErrOrderLimitNotFound,         // Is thrown when the order limit isn't found for a particular exchange, asset, pair
 	limits.ErrEmptyLevels,                // Is thrown if limits are not provided for the asset
+	limits.ErrPriceBelowMin,              // Is thrown when the price is below the minimum limit for an exchange
+	limits.ErrPriceExceedsMax,            // Is thrown when the price exceeds the maximum limit for an exchange
+	limits.ErrPriceExceedsStep,           // Is thrown when the price doesn't conform to the step increment for an exchange
+	limits.ErrAmountBelowMin,             // Is thrown when the amount is below the minimum limit for an exchange
+	limits.ErrAmountExceedsMax,           // Is thrown when the amount exceeds the maximum limit for an exchange
+	limits.ErrAmountExceedsStep,          // Is thrown when the amount doesn't conform to the step increment for an exchange
+	limits.ErrNotionalValue,              // Is thrown when the notional value is below the minimum for an exchange
```

Comment:
🐒 Optional nitpick and out of scope for this PR, but:
This error name is not clear enough.
It's the only one here that actually needs a comment, and that's because it doesn't fully qualify the error.